cleanlab/object_detection/filter.py

```diff
@@ -217,3 +234,146 @@ def _pool_box_scores_per_image(is_issue_per_box: List[np.ndarray]) -> np.ndarray
         if np.sum(issue_per_box) > 0:
             is_issue[idx] = 1
     return is_issue
+
+
+def _process_class_list(class_list: List[np.ndarray], class_dict: Dict[int, float]):
+    """
+    Converts a list of classes using a float dictionary into a list where each class is replaced by its corresponding float value.
```

Comment:
Isn't `class_list` a list of lists of classes?

Reply:
we can't iterate here: thr_classes is a list[list]]
```
issue_per_box = []
            for i in range(len(score_per_box)):
                issue_per_box.append(score_per_box[i] <= thr_classes[idx][i] * threshold_factor)
```

Reply:
labels[0]['labels'] = array([3, 0, 0, 1])
it's passed as np.array itself

Reply:
Can you clarify what the input/output of this function is better in the docstring. is `class_list`a list of classes or what is the exact format of it